src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/throwscount/InputThrowsCountMaxAllowZero.java

```diff
@@ -13,7 +13,7 @@
 import java.io.FileNotFoundException;
 import java.sql.SQLException;
 
-public class InputThrowsCount4 {
+public class InputThrowsCountMaxAllowZero {
```

Comment:
File name should be charged.
Strange that CI is not failing.

Reply:
@romani ,

The filename has already been changed and hence the successful CI..
InputThrowsCount4 -->  `InputThrowsCountMaxAllowZero`

I'm not sure what you are expecting. Could you please elaborate ?

Reply:
> I'm not sure what you are expecting. Could you please elaborate ?
> 
> Also, the failing CI no-error-xwiki is now resolved through my merged PR https://github.com/xwiki/xwiki-platform/pull/2934

@romani , ping please